regression_test.py

```diff
@@ -115,13 +118,16 @@ def run_widelands(self, wlargs, which_time):
                 stdout_file.write(bytes_to_str(line))
                 stdout_file.flush()
```

Comment:
Essentially this writes stdout to the file, right? So it should be possible to give `stdout_file` directly to `Popen`. Maybe one of `encoding` or `text` has to be set also to handle the encoding.
Then you can remove the infinite while loop and use the timeout in `communicate` instead.

Reply:
:+1:  Thanks for the tip. :smiley: I'll wait for the current test run to finish (to collect some more data on timeouts and running times), then try to do it.

Reply:
Thank you again. It works for me without any other options to Popen(), let's see in the workflow run.